src/lib/cli/sonarrc-generator.ts

```diff
@@ -59,9 +59,9 @@ export const initSonarrc = async () => {
 
     const questions = [
         {
-            choices: collectorKeys,
-            message: 'What collector do you want to use?',
-            name: 'collector',
+            choices: connectorKeys,
+            message: 'What connector do you want to use?',
```

Comment:
We could improve already the question here and briefly explain what a connector is. Maybe add a description for each one? Something similar to:
> What connector do you want to load the content of the website?
> * cdp (you will need to have chrome installed)
> * jsdom (will use jsdom, no additional installation required)